katello-installer/katello-installer-base.spec

```diff
@@ -2,7 +2,7 @@
 %undefine scl_prefix
 %global scl_ruby /usr/bin/ruby
 
-%global prever .rc1
+%global prever .rc1.1
```

Comment:
What about having a release variable? I think that's what you did for other uses of the rc flag, and that keeps things a bit more obvious for those use to thinking of release numbers especially when we get to GA?